### PR TITLE
Ensure a wrong running directory will not hinder the project from building

### PR DESCRIPTION
**Pull request by SinTh0r4s**
Resolve Issue #43

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -487,20 +487,20 @@ static URL availableBuildScriptUrl() {
...
     }
     return false
 }
 
-static boolean isNewBuildScriptVersionAvailable() {
+static boolean isNewBuildScriptVersionAvailable(String projectDir) {
     Map parameters = ["connectTimeout": 2000, "readTimeout": 2000]
 
-    String currentBuildScript = new File("build.gradle").getText()
+    String currentBuildScript = new File(projectDir, "build.gradle").getText()
```
Comment (TheElan) [inline build.gradle:503]: what do you think about creating function which would do this for us, like `def projectFile(String name) = new File(projectDir, name)` or smth like that?

Comment (SinTh0r4s) [reply build.gradle:503]: Certainly a good thing! But is `projectDir` available in a function? Gonna test later

File: build.gradle [hunk 2]
```diff
@@ -93,27 +93,31 @@ def checkPropertyExists(String propertyName) {
...
+}
+
 
 String javaSourceDir = "src/main/java/"
 String scalaSourceDir = "src/main/scala/"
 
 String targetPackageJava = javaSourceDir + modGroup.toString().replaceAll("\\.", "/")
 String targetPackageScala = scalaSourceDir + modGroup.toString().replaceAll("\\.", "/")
-if((new File(projectDir, targetPackageJava).exists() || new File(projectDir, targetPackageScala).exists()) == false) {
+if((getFile(targetPackageJava).exists() || getFile(targetPackageScala).exists()) == false) {
```
[the remark this answers is not in the document]
Comment (TheElan) [reply build.gradle:106]: also I think you can have functions below invocation (as in using top-down approach)

**Review by TheElan — APPROVED** [review]
Had one last suggestion, other then that it looks fine, up to you if you want to do anything more.

**Pull request merged**